docs/source/configuration/volto-slate/configuration-settings.md

```diff
@@ -245,12 +245,12 @@ slate.runtimeDecorators = [([node, path], ranges) => ranges];
 
 The setting `slate.useLinkedHeadings` controls whether `volto-slate` creates anchors for headings, such as `h1` and `h2`, in the editor.
 
-The default setting is `true`.
+The default setting is `false`.
```

Comment:
This doesn't match what I see in the code.

It creates anchor links unless the setting is explicitly equal to `false`:
https://github.com/plone/volto/blob/0deb876b34bcd698dc46282884ce5417aeae0fce/packages/volto-slate/src/editor/render.jsx#L179

So if useLinkedHeadings isn't set, it has the same effect as setting it to true.

Reply:
> This doesn't match what I see in the code.
> 
> It creates anchor links unless the setting is explicitly equal to `false`:
> 
> https://github.com/plone/volto/blob/0deb876b34bcd698dc46282884ce5417aeae0fce/packages/volto-slate/src/editor/render.jsx#L179
> 
> So if useLinkedHeadings isn't set, it has the same effect as setting it to true.

@davisagli you are correct the setting is missing, and therefore it is opt-out.
I would add a note that mentions in the documentation for this setting that it will add the anchors only
for logged in users after whatever version made those changes.
What do you think about my idea, would you add this info somewhere else?

Reply:
@ichim-david yeah, it makes sense to clarify that the feature is only for editors